eslint.common.config.js

```diff
@@ -0,0 +1,100 @@
+import jsdoc from "eslint-plugin-jsdoc";
+import ava from "eslint-plugin-ava";
+import globals from "globals";
+import js from "@eslint/js";
+import google from "eslint-config-google";
+
+export default [{
+	ignores: [ // Common ignore patterns across all tooling repos
+		"**/coverage/",
+		"test/tmp/",
+		"test/expected/",
+		"test/fixtures/",
+		"**/docs/",
+		"**/jsdocs/",
+	],
+}, js.configs.recommended, google, ava.configs["flat/recommended"], {
+	name: "Common ESLint config used for all tooling repos",
+
+	plugins: {
+		jsdoc,
+		ava,
```

Comment:
I don't think `ava` is actually needed here. 
The `ava.configs["flat/recommended"]` has already packed this